integration-tests/actions/vrf/vrfv2plus/actions.go

```diff
@@ -0,0 +1,62 @@
+package vrfv2plus
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"sync"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+)
+
+func WaitForRequestCountEqualToFulfilmentCount(
```

Comment:
why this file was added? we should use method from [here](https://github.com/smartcontractkit/chainlink/blob/9d6c894ade1d38fb8312552b6a03836b7218136d/integration-tests/actions/vrf/common/actions.go#L149)

Reply:
Can't. Because the metrics is a new struct due to the changes. Unless we want to add the same changes to the old V2 contracts. Or add a new field to the current struct just that it wouldn't be populated. It's gonna be messy.

Reply:
but now its even more messier with duplicates. I mean, I have changed the VRF v2 as well and now it should be fine